### PR TITLE
Add new styling configurations

### PR DESCRIPTION
**Pull request by handotdev**
Add the new layout configurations. Learn more at https://www.notion.so/mintlify/New-mint-json-theme-customizations-05e3f8cd744844eaa847e29c0f9382f7?pvs=4

### CODE REVIEW

File: settings/global.mdx [hunk 1]
```diff
@@ -66,15 +66,29 @@ settings. Learn more about the [properties](#properties) or from an
...
+  A preset theme configuration that changes the look and feel of the project. A theme is a set of default styling configurations.
+  
+  Examples:
   [Venus](https://starter-venus.mintlify.app),
   [Quill](https://starter-quill.mintlify.app),
   [Prism](https://starter-prism.mintlify.app)
 </ResponseField>
 
+<ResponseField name="layout" type={'"topnav" | "sidenav" | "solidSidenav"'} default="topnav">
+  The layout of the project. Setting it to `topnav` will feature the top bar as the primary navigation. Setting it to `sidenav` will feature the sidebar. The `solidSidenav` will have an active sidenav.
```
Comment (hahnbeelee) [inline settings/global.mdx]: im having trouble understanding this

File: settings/global.mdx [hunk 2]
```diff
@@ -66,15 +66,29 @@ settings. Learn more about the [properties](#properties) or from an
...
+</ResponseField>
+
+<ResponseField name="background" type="Background">
+  Set a decorative background
+  <Expandable title="Background">
+    <ResponseField name="styling" type={'"gradient" | "grid" | "windows"'}>
+      The style of the decorative background
```
Comment (hahnbeelee) [inline settings/global.mdx]: i think we should maybe have links to examples for this.

File: settings/global.mdx [hunk 3]
```diff
@@ -88,26 +102,52 @@ settings. Learn more about the [properties](#properties) or from an
...
+}
+```
+
       </CodeGroup>
+    </ResponseField>
+  </Expandable>
+</ResponseField>
 
+<ResponseField name="sidebar" type="Sidebar">
+  Styling configurations for components of the sidebar
```
Comment (hahnbeelee) [inline settings/global.mdx]: this one is also hard to understand

File: settings/global.mdx (hunk 2 above)
Comment (handotdev) [reply settings/global.mdx]: Don't have a ton of examples atm. We can build examples and then link back here

**Review by hahnbeelee — APPROVED** [review]

**Pull request merged**